src/standard_name.jl

```diff
@@ -0,0 +1,208 @@
+        @optic(_.land.demand.paddy.parameters.irrigation_trigger),
+    "land~irrigated-paddy__gross_water_demand_volume_flux" =>
+        @optic(_.land.demand.paddy.variables.demand_gross),
+    "land~irrigated-non-paddy__irrigation_trigger_flag" =>
+        @optic(_.land.demand.nonpaddy.parameters.irrigation_trigger),
+    "land~irrigated-non-paddy__gross_water_demand_volume_flux" =>
+        @optic(_.land.demand.nonpaddy.variables.demand_gross),
+    "land~irrigated__allocated_water_volume_flux" =>
+        @optic(_.land.allocation.variables.irri_alloc),
+    "land~non-irrigated__allocated_water_volume_flux" =>
+        @optic(_.land.allocation.variables.nonirri_alloc),
+    "subsurface_water_abstraction__volume_flux" =>
+        @optic(_.land.allocation.variables.act_groundwater_abst),
+    "subsurface_water__hydraulic_head" =>
+        @optic(_.routing.subsurface_flow.aquifer.variables.head),
+    "subsurface_water_sat-zone_top__net_recharge_volume_flux" =>
+        @optic(_.routing.subsurface_flow.boundaries.recharge.variables.flux),
+    "land_drain_water~to-subsurface__volume_flow_rate" =>
+        @optic(_.routing.subsurface_flow.boundaries.drain.variables.flux),
+    "river_water~to-subsurface__volume_flow_rate" =>
+        @optic(_.routing.subsurface_flow.boundaries.river.variables.flux),
+)
+
+const sediment_standard_name_map = Dict{String, ComposedFunction}(
+    "atmosphere_water__precipitation_volume_flux" =>
+        @optic(_.land.atmospheric_forcing.precipitation),
+    "river_water__volume_flow_rate" =>
+        @optic(_.routing.river_flow.hydrological_forcing.q_river),
+    "river_water__depth" =>
+        @optic(_.routing.river_flow.hydrological_forcing.waterlevel_river),
+    "land_surface_water__volume_flow_rate" =>
+        @optic(_.land.hydrological_forcing.q_land),
+    "land_surface_water__depth" => @optic(_.land.hydrological_forcing.waterlevel_land),
+    "vegetation_canopy_water__interception_volume_flux" =>
+        @optic(_.land.hydrological_forcing.interception),
+    "soil_erosion~rainfall__mass_flow_rate" =>
+        @optic(_.land.rainfall_erosion.variables.amount),
+    "soil_erosion~overland_flow__mass_flow_rate" =>
+        @optic(_.land.overland_flow_erosion.variables.amount),
+    "soil_erosion__mass_flow_rate" => @optic(_.land.soil_erosion.variables.amount),
+    "soil_erosion_clay__mass_flow_rate" => @optic(_.land.soil_erosion.variables.clay),
+    "land_surface_water_sediment_transport_capacity__mass_flow_rate" =>
+        @optic(_.routing.overland_flow.transport_capacity.variables.amount),
+    "land_surface_water_clay_transport_capacity__mass_flow_rate" =>
+        @optic(_.routing.overland_flow.transport_capacity.variables.clay),
+    "land_surface_water_sediment~to-river__mass_flow_rate" =>
+        @optic(_.routing.overland_flow.to_river.variables.amount),
+    "land_surface_water_clay~to-river__mass_flow_rate" =>
+        @optic(_.routing.overland_flow.to_river.variables.clay),
```

Comment:
This one is needed for all particle class clay/silt/sand/sagg/lagg
```suggestion
    "land_surface_water_clay~to-river__mass_flow_rate" =>
        @optic(_.routing.overland_flow.to_river.variables.clay),
    "land_surface_water_silt~to-river__mass_flow_rate" =>
        @optic(_.routing.overland_flow.to_river.variables.silt),
    "land_surface_water_sand~to-river__mass_flow_rate" =>
        @optic(_.routing.overland_flow.to_river.variables.sand),
    "land_surface_water_aggregates~small~to-river__mass_flow_rate" =>
        @optic(_.routing.overland_flow.to_river.variables.sagg),
    "land_surface_water_aggregates~large~to-river__mass_flow_rate" =>
        @optic(_.routing.overland_flow.to_river.variables.lagg),
```

Reply:
And these output parameters are also required for standard use of the sediment model? 

For expert users we could also opt to remove these parameters and an user can then use then the previous implementation (with dot notation).